scripts/verify-release

```diff
@@ -18,17 +18,16 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Runs automated checks of website content, first building website content
-# and then re-generating latest API / CLI docs.
-
 # Should be run in CI to verify changes.
+# Run by a prow job which we plan to remove.
+# * https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/website/cert-manager-website-presubmits.yaml
+#
+# TODO(wallrj): Remove this when the prow job has been removed
 
 REPO_ROOT="${REPO_ROOT:-$(cd "$(dirname "$0")/.." && pwd)}"
 
-cd "${REPO_ROOT}"
-
-echo "+++ Building site fully"
-"${REPO_ROOT}/scripts/build-release"
```

Comment:
It's not necessary for prow to build the site fully.
It's slow and is already executed by the netlify preview which will soon become a required check on all website PRs:
 * https://github.com/jetstack/testing/pull/788